CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 0.9.0
+
+* Changelog started, changes from previous release `0.8.0`:
+
+* Major breaking:
+  * Removed instances due to migration to `haskeline >= 0.8 && < 0.9`:
+    * `instance MonadException m => MonadException(StateT(HashMap FilePath NExprLoc) m)`
+    * `instance MonadException m => MonadException(Fix1T StandardTF m)`
+
+* Minor:
+  * Development package neetly exposes majority of `Nixpkgs haskell.lib` features
+  * Fixed SDist build
+  * Changes in some REPL funtion types, including `main` function
+  * Freed from `cryptohash-{md5,sha1,sha256,sha512}` dependencies in favour of `hashing` for the time being
+  * New CI:
+    * Provides `Nixpkgs haskell.lib` features
+    * SDist build
+    * Strict build (compilation with `-Werror`) for `GHC < 8.8`
```

Comment:
It is just project state, quality news, news for devs, maybe then we can just remove `Minor`, since dependencies on `cryptohash-{md5,sha1,sha256,sha512}` are also not user-facing changes.

Reply:
I wrote it more explicitly to get a feeling of what can be slashed on next recurses.

I feel like most of `minor` changes in the project can be skipped, and only `major` and `medium` of extensions in API functionality should be really mentioned.

Reply:
> dependencies on `cryptohash-{md5,sha1,sha256,sha512}` are also not user-facing changes.

I think that's somewhat user-facing, since their dependency trees change.

Reply:
Then we may argue that everything is user-facing, since user uses the project code and dependencies of the project.

Did you ever cared when unknown to you Haskell libraries changed lib dependencies or introduced custom prelude, in libraries/modules you never looked in the code of?